sdk/python/feast/feature_store.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 The Feast Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional
+from feast.feature_store_config import Config
+
+
+class FeatureStore:
```

Comment:
I think we should add a comment to the FeatureStore class.

Reply:
I'm not sure how useful `fat client` is to users. How about `A FeatureStore object is used to define, create, and retrieve features.`